File: client/webflix/src/app/model/wishlist-summary.ts
import {Movie} from './movie';

export class WishlistSummary {
  movies: Movie[];
  total: number;

  static empty(): WishlistSummary {
    let empty = new WishlistSummary();
    empty.movies = [];
    empty.total = 0;
    return empty;
  }
}

File: server/webflix/src/main/java/com/instil/webflix/wishlist/model/WishlistSummary.java
package com.instil.webflix.wishlist.model;

import java.math.BigDecimal;

import com.instil.webflix.movies.model.Movie;

public class WishlistSummary {
	private Iterable<Movie> movies;
	private BigDecimal total;

	public WishlistSummary(Iterable<Movie> movies, BigDecimal total) {
		this.movies = movies;
		this.total = total;
	}

	public Iterable<Movie> getMovies() {
		return movies;
	}
	
	public BigDecimal getTotal() {
		return total;
	}
}